modules/reviewing-the-installation-log.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="reviewing-the-installation-log_{context}"]
+= Reviewing the installation log
+
+You can review a summary of an installation in the {product-title} installation log. If an installation succeeds, the information required to access the cluster is included in the log.
+
+.Prerequisites
+
+* You have access to the installation host.
+
+.Procedure
+
+* Review the `.openshift_install.log` log file in the installation directory on your installation host:
++
+[source,terminal]
+----
+$ cat <install_dir>/.openshift_install.log
+----
++
+.Example output
++
+Cluster credentials are included at the end of the log if the installation is successful, as outlined in the following example:
++
+[source,terminal]
+----
+...
+time="2020-12-03T09:50:47Z" level=info msg="Install complete!"
+time="2020-12-03T09:50:47Z" level=info msg="To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=<install_dir>/auth/kubeconfig'"
+time="2020-12-03T09:50:47Z" level=info msg="Access the OpenShift web-console here: https://console-openshift-console.apps.<cluster_name>.<base_domain>"
+time="2020-12-03T09:50:47Z" level=info msg="Login to the console with user: \"kubeadmin\", and password: \"<password>\""
```

Comment:
I don't know if saying anything about `<password>` is needed here or not. I assume it's replaced with a random password, generated by OCP?

Reply:
Thank you for highlighting this. I have subsequently updated the example output to include an example password. I have also replaced `<cluster_name>.<base_domain>` with `mycluster.example.com`. I noticed that `example.com` is used as an example base domain elsewhere in the installation book.